batch/file-ingest/src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 spring.application.name=fileIngest
 spring.datasource.initialization-mode=always
+spring.datasource.url=jdbc:mysql://mysql:3306/dataflow
```

Comment:
are these values actually needed in the properties file or might this be left over from debugging? in for example k8s or docker compose, these values are passed already via vars in their respective config files. id assume on CF there may be some auto-configuration or whatnot

since we didn't have this before, can you verify this is actually needed now across the supported platforms?